src/scratchtocatrobat/converter/converter.py

```diff
@@ -1089,7 +1089,7 @@ def _add_default_behaviour_to(sprite, sprite_context, catrobat_scene, catrobat_p
 
         object_relative_scale = scratch_object.get_scale() or 1
         if costume_resolution is not None:
-            object_scale = object_relative_scale * 100.0 / costume_resolution
+            object_scale = object_relative_scale * 100.0
```

Comment:
this was responsible for wrong object scaling/ that we "sometimes" had wrong scaling with the set size to 50% block at start (and sometimes not)